iqoptionapi/iqoptionapi
# Fix typos in spanish docs

## Issue Description

This PR is created to fix a few typos found while checking the Spanish docs.

In here I'm not fixing all of the typos but the idea is that while I'm checking the docs I can open more PR to fix the new typos found

**REVIEWERS NOTE:**
The best way to review this PR is going commit by commit

In the end, once this PR is ready to be merged, we can consider doing a squash merge to keep a cleaner commit history (just one commit with all PR changes) but keeping a reference to all the commits created on this PR

## Proposed Change

N/A

Make it easy to code review

- Replace 'Soporta' with 'Apoya'
- Replace 'Furutos de divisas' with 'Futuros de divisas'
- Replace 'Table of contents' with 'Tabla de contenidos'

**NOTE:**
You'll notice that changes in file `docs/search/search_index.json` are basically impossible to review here. I'd suggest you do any of these things
- Compare both files using `diff` in your terminal
- Compare both files using a web page to find diffs in texts

You should find that the only difference is `Futuros de divisas` instead of `Furutos de divisas`


## Tested code evidence

![image](https://user-images.githubusercontent.com/11419017/133153917-2721d7d0-0b54-4b32-a429-8ce555b84a8c.png)
![image](https://user-images.githubusercontent.com/11419017/133153959-dd1ce94e-4729-463b-bd71-b443913bf2b9.png)
![image](https://user-images.githubusercontent.com/11419017/133154029-1e92cfde-4aee-48dc-8ebe-b8d30ccd036b.png)


## How to test code

- Go to the docs (in Spanish) https://iqoptionapi.github.io/iqoptionapi/es
- Start navigating to the next section from the page you are in
- You'll be able to see all the new changes
